test/Makefile

```diff
@@ -0,0 +1,33 @@
+.PHONY: default all
+default: test-kernel-config
+all: default
+
+MOBY?=../bin/moby
+
+define check_test_log
+	@cat $1 |grep -q 'Moby test suite PASSED'
```

Comment:
this changed with: https://github.com/linuxkit/linuxkit/pull/1635
please update